cjs-builder/index.ts

```diff
@@ -28,7 +48,8 @@
 	const options: Parameters<typeof esbuild.build>[0] = {
 		entryPoints: [relativeInputFilePath],
 		bundle: true,
-		packages: 'external',
+		// packages: 'external',
+		external: getExternal(packageJSON),
```

Comment:
actually we might pass the external package names as the yargs options to the CJS builder OR
We actually remove both `packages` and `external` and allow `esbuild` to bundle the normal dependencies. We will have to move all `dependencies` to `devDependencies`

Reply:
let do this one
>We actually remove both packages and external and allow esbuild to bundle the normal dependencies. We will have to move all dependencies to devDependencies